apps/profiles/upstream/base/kustomization.yaml

```diff
@@ -16,5 +16,7 @@ images:
 
 configMapGenerator:
 - name: namespace-labels-data
+  options:
```

Comment:
You are not allowed to change stuff in the upstream folder. Please try without it and you will find a way.

Reply:
The documentation does day that _merge_ behavior should add or update values to the configMap.
`https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/configmapgenerator/#overriding-base-configmap-values:~:text=merge%3A%20add%20or%20update%20the%20values%20in%20an%20existing%20ConfigMap%20from%20the%20base.`

I also found an issue here at `https://github.com/kubernetes-sigs/kustomize/issues/370#:~:text=%40mr%2Dkaran%20%40anshul,here%20as%20well.` which also like us, wishes the configMap to get configured with merge behavior. But, I guess that's why I see the same in cluster while trying to merge the configMap with new labels. There's a new configMap that gets created with a new suffix to which the workloads point to (as they should) but it doesn't include the values of previous configMap due to the hash suffix.